src/database.ts

```diff
@@ -114,25 +114,24 @@ export const redshiftClusterMaintenanceSettings: ResourceValidationPolicy = {
             },
             preferredMaintenanceWindow: {
                 type: "string",
-                default: "Mon:09:30-Mon:10:00",
             },
             automatedSnapshotRetentionPeriod: {
                 type: "number",
-                default: 1,
```

Comment:
Similarly, we don't want a default value for this one. If it is specified, then we'll do further checks.

I think the "If undefined, the default is 1." in the policy validation handler is referring to if it is undefined on the cluster, it means its 1 for the cluster.